optuna/samplers/_tpe/sampler.py

```diff
@@ -920,16 +957,24 @@ def _get_observation_pairs(
 
 
 def _get_multivariate_observation_pairs(
-    study: Study, param_names: List[str]
+    study: Study,
```

Comment:
This function is merged in #2618. I think it's good to wait for #2618 to merge this PR.